backend/server.py

```diff
@@ -37,19 +39,27 @@ def api(app_name, api_call):
 Database State:
 {db[app_name]["state"]}
 
-Output the API response prefixed with 'API response:'. Then output the new database state as json, prefixed with 'New Database State:'. If the API call is only requesting data, then don't change the database state, but base your 'API Response' off what's in the database.
+Output the API response as json prefixed with '!API response!:'. Then output the new database state as json, prefixed with '!New Database State!:'. If the API call is only requesting data, then don't change the database state, but base your 'API Response' off what's in the database.
 """
     completion = ray.get(gpt3.remote(gpt3_input))
```

Comment:
now that it's one llm call, can you remove all usages of `ray` library

Reply:
I have removed ray in `server.py` and `requirements.txt`.